server/datastore/mysql/migrations/tables/20231016091915_FixQueriesWithEmptyLoggingType.go

```diff
@@ -0,0 +1,27 @@
+package tables
+
+import (
+	"database/sql"
+	"fmt"
+
+	"github.com/fleetdm/fleet/v4/server/fleet"
+)
+
+func init() {
+	MigrationClient.AddMigration(Up_20231016091915, Down_20231016091915)
+}
+
+func Up_20231016091915(tx *sql.Tx) error {
+	_, err := tx.Exec(`
+		UPDATE queries SET logging_type = ? WHERE logging_type = '';
+    `, fleet.LoggingSnapshot)
+	if err != nil {
+		return fmt.Errorf("failed to create table query_results: %w", err)
```

Comment:
Should probably update with `queries` table name instead of `query_results`

Reply:
Damn copy paste! ([Fix copy paste error string](https://github.com/fleetdm/fleet/pull/14575/commits/7765b43ea4a57a68a367b1c9212413506e11470f))